cli/ioctl/cmd/accountcreate.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2018 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"strings"
+	"fmt"
+
+	"github.com/iotexproject/go-ethereum/crypto"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+
+	"github.com/iotexproject/iotex-core/pkg/keypair"
+	"github.com/iotexproject/iotex-core/pkg/log"
+	"github.com/iotexproject/iotex-core/address"
+)
+
+var numAccounts int
```

Comment:
if we put all commands under `cmd`, this package will be flooded with all different global variables.

Reply:
Any suggestion? Honestly, I don't worry about this for now as we have few commands for each resource.

Reply:
@raullenchai This seems to be overlap with #132 . If we can list cli needed and group them in different groups, we can organise code as per suggestion written in #132 and #21

Reply:
you can add another layer of package, for example: https://github.com/kubernetes/kubernetes/tree/master/pkg/kubectl/cmdA

Reply:
Done